test/inductor/test_flex_attention.py

```diff
@@ -120,6 +120,13 @@ def _trig2(score, b, h, m, n):
 S = 2048
 D = 64
 
+test_input_strides = [
```

Comment:
You can move this closer to the actual test definition, since this is only used in one place.

Reply:
Also, you might want to write this in terms of B, H, S, and D.

Reply:
Fixed code style 
 https://github.com/pytorch/pytorch/pull/128915/commits/b345ae6dc34af6393ef88a68dd886ff5f8206b9a